src/test/MoveTest.java

```diff
@@ -0,0 +1,37 @@
+import Board.Tile;
+import Enums.Colors;
+import Pieces.Bishop;
+import Pieces.Piece;
+import Pieces.Rook;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class MoveTest {
+    Tile source = new Tile(1,1);
+    Tile destination = new Tile(2,2);
+    Piece movedPiece = new Rook(Colors.BLACK, source);
+    Piece capturedPiece = new Bishop(Colors.WHITE, destination);
+    @Test
+    public void moveTest() throws Exception {
+        Move move = new Move(source,destination,movedPiece,null);
+        assertEquals(Colors.BLACK, movedPiece.getPieceColor());
```

Comment:
Is this a test for the constructor? If so, you're testing the fields on the movedPiece/CapturedPiece and not the Move object

Reply:
Also, if it is a test for the constructor, you should probably test the 4 fields that are set in the constructor, and not the color